client/my-sites/earn/memberships/style.scss

```diff
@@ -240,6 +240,32 @@
 	float: right;
 }
 
+.memberships__dialog-nav.section-nav {
+	box-shadow: none;
+	border-bottom: 1px solid var( --color-neutral-5 );
+}
+
+.memberships__dialog-sections {
+	display: grid;
+	grid-template: 'item' 1fr / 1fr;
+}
+
+.memberships__dialog-section {
+	grid-area: item; // Sections are displayed in the same grid area so the dialog takes the size of the biggest section.
+	background-color: var( --color-surface );
+	z-index: 0;
```

Comment:
If we need a z-index value let's add this to https://github.com/Automattic/wp-calypso/blob/master/client/assets/stylesheets/shared/functions/_z-index.scss

Reply:
7c366dd